resources/tsx/catch.tsx

```diff
@@ -0,0 +1,24 @@
+
+const parsePromises = new Set<Promise<string | null>>();
+
+parsePromises.add(
+    fetchData("www.google.com").catch((reason) => { //+0 complexity
```

Comment:
Good point that we do not count them as well currently. We should document that behaviour somewhere for the user, also that we do not count things like the build-in `forEach` function. 

For now, I have added that point to the list in issue #88 to may discuss later if we should change something about it.